webdriver/tests/support/fixtures_bidi.py

```diff
@@ -18,11 +18,12 @@
 async def add_preload_script(bidi_session):
     preload_scripts_ids = []
 
-    async def add_preload_script(function_declaration, arguments=None, sandbox=None):
+    async def add_preload_script(function_declaration, arguments=None, sandbox=None, contexts=None):
         script = await bidi_session.script.add_preload_script(
             function_declaration=function_declaration,
             arguments=arguments,
             sandbox=sandbox,
+            contexts=contexts,
```

Comment:
I cannot find the update for the BiDi client at `tools/webdriver/webdriver/bidi/modules/script.py` so this is basically ignored.

Maybe this is why [all the Chrome tests are failing](https://github.com/web-platform-tests/wpt/pull/41932/checks?check_run_id=16786694756)?